iml-system-test-utils/src/lib.rs

```diff
@@ -126,3 +129,35 @@ pub fn get_local_server_names<'a>(servers: &'a [&'a str]) -> Vec<String> {
         .map(move |x| format!("{}.local", x))
         .collect()
 }
+
+#[async_trait]
+pub trait CmdErrSos {
+    async fn handle_test_result(self, hosts: &[&str], prefix: &str) -> Result<(), CmdError>;
```

Comment:
Can we use generics here for the err type?

Then we wouldn't need two separate traits / impls.

Reply:
Fixed.